templates/hooks/archive/sdk_update_pre_build_request.go.tpl

```diff
@@ -0,0 +1,19 @@
+if archiveInTerminalState(latest) {
+	msg := "Archive is in '" + *latest.ko.Status.State + "' status"
```

Comment:
nit: Move these messages into constants in `hook.go` and use `fmt.Sprintf`

Reply:
Will change. Was following other controllers where this is a pattern and thought this is to avoid `fmt` allocations :D

Reply:
Changed the wording using `fmt.Sprintf` but kept it in the same location because there was no more readable way IMHO to express these messages by moving to separate file.